src/styles/utiles/__normalize.scss

```diff
@@ -0,0 +1,6 @@
+* {
+  cursor: pointer;
+  position: relative;
+  box-sizing: border-box;
```

Comment:
it meant move only `box-sizing: border-box;` to `*`

no need to apply `cursor: pointer;
  position: relative;` for all elements